Database/Corrections/sodItemFixes.lua

```diff
@@ -32,6 +32,12 @@ function SeasonOfDiscovery:LoadItems()
         [209846] = {
             [itemKeys.objectDrops] = {409692},
         },
+        [210187] = {
+            [itemKeys.objectDrops] = nil,
+            [itemKeys.itemDrops] = nil,
+            [itemKeys.vendors] = nil,
```

Comment:
And these can be removed (since we have them in the base file)

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5339/commits/b76ce8abcbab8297781523dc87d52c9a739b652f